handlers.go

```diff
@@ -35,7 +35,7 @@ func (a *Application) home(w http.ResponseWriter, r *http.Request) {
 
 	var ok bool
 
-	data.PageTemplate, ok = a.templateCache["home.page.tmpl"]
+	data.PageTemplate, ok = s.templateCache["home.page.tmpl"]
 	if !ok {
 		fmt.Fprintf(w, fmt.Sprint("template does not exist: home.page.tmpl"))
```

Comment:
Is this possible? (Let me rephrase the question: could we _make_ this impossible? After all, we control the map. Suppose we defined some constant representing the home page template?)